### PR TITLE
Use light html5test page for first run

### PR DESCRIPTION
**Pull request by dzedro**
any page is better and easier to load than suse.com

- Related ticket: https://progress.opensuse.org/issues/40637
- Needles: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/946
- Verification run: http://10.100.12.155/tests/7129

### CODE REVIEW

**Comment by coolo** [comment]
Aehm, all your firefox tests in the verification link use suse.com?

**Comment by coolo** [comment]
The stall is also a sign that you guys collected way too many firefox needles over time :(

**Comment by dzedro** [comment]
@coolo I will cleanup also needles, but the problem is also low RAM, this firefox tests on SLE15 has QEMURAM=2048, but this was lost when the old backup was restored ^^

**Comment by coolo** [comment]
You was in CC when I asked to re-review the rollback for recent changes :)

**Comment by coolo** [comment]
But the stall is happening on the worker, not in the SUT, so it doesn't really explain this.

**Comment by dzedro** [comment]
Ye I added it back on test where I knew I added it day before, but this I forgot :(

File: lib/x11test.pm [hunk 1]
```diff
@@ -521,7 +521,7 @@ sub start_clean_firefox {
 
     x11_start_program('xterm');
     # Clean and Start Firefox
-    type_string "killall -9 firefox;rm -rf .moz* .config/iced* .cache/iced* .local/share/gnome-shell/extensions/*; firefox suse.com >firefox.log 2>&1 &\n";
+    type_string "killall -9 firefox;rm -rf .moz* .config/iced* .cache/iced* .local/share/gnome-shell/extensions/*; firefox html5test.opensuse.org >firefox.log 2>&1 &\n";
```
Comment (okurz) [inline lib/x11test.pm]: Even more lightweight would be http://static.opensuse.org/ when we do not want to test the rendering of the html5 page itself. How about using html5test in a single test module and static in all others?

Comment (dzedro) [reply lib/x11test.pm]: it has to trigger tracking pop-up

Comment (dzedro) [reply lib/x11test.pm]: but http://static.opensuse.org could be used as homepage, I will try to use it

**Pull request merged**